hack/install.sh

```diff
@@ -529,6 +529,19 @@ do_install() {
 	exit 1
 }
 
+while getopts "e:" opt; do
+	case $opt in
+		e)
+			environment=$OPTARG
+			;;
+	esac
+done
+
+if [ ! -z "$environment" ] && [ $environment = "AzureChinaCloud" ]; then
+	apt_url="https://mirror.azure.cn/docker-engine/apt"
+	yum_url="https://mirror.azure.cn/docker-engine/yum"
+fi
```

Comment:
I think it'd make sense to move this whole block up closer to the initial setting of these variables (so we keep our constants all in one place, especially including the new args to the script this introduces).